packages/www/components/Dashboard/Chart/index.tsx

```diff
@@ -24,16 +39,29 @@ const CustomTooltip = ({ active, payload }: any) => {
             fontSize: "12px",
             color: "white",
           }}>
-          Rate: <b>{payload[0].value} kbps</b>
+          {payload.map((item) => (
+            <div>
+              {item.dataKey}: <b>{item.value} kbps</b>
+            </div>
+          ))}
         </Text>
       </Box>
     );
   }
 
   return null;
 };
-
-const Chart = ({ data }) => {
+const Chart = ({
+  data,
+  multiData,
+}: {
+  data: Array<{ name: number; "Session bitrate": number }>;
+  multiData?: Array<{
+    [kbps: string]: number;
```

Comment:
Isn't the key the multistream name here?
```suggestion
    [name: string]: number;
```